### ISSUE TITLE
No need to resize input image

### ISSUE THREAD

**Issue by MasKogNavier** [yumingj/C2-Matching#22]
I tried problem # 10 because I don't need to resize the input image, but it doesn't work.
And i tried commented out the PIL.resize part
ex) l.129 in `ref_cufed_dataset.py`  
before:
`img_in_lq = img_in_pil.resize((lq_w, lq_h), Image.BICUBIC)
`
after:
`img_in_lq = img_in_pil#.resize((lq_w, lq_h), Image.BICUBIC)
`
I got below error.(input and ref image size are 128x128)

`RuntimeError: invalid argument 0: Sizes of tensors must match except in dimension 1. Got 32 and 128 in dimension 2 at /opt/conda/conda-bld/pytorch_1579040055865/work/aten/src/THC/generic/THCTensorMath.cu:71
`
How do I use an input image of my own defined size(no need to resize image)?

**Comment by yumingj** [comment]
Hi, thanks for your interest in our work!

Can you have a look at the size of img_in_up? It should have the same size as img_ref_up.

**Comment by MasKogNavier** [comment]
thank you for your reply.
With the current settings, I intend to super-resolution 128x128 input image (img_in_lr) to x4.
As a confirmation, if the output is 512x512, does it mean that 512x512 should be prepared for img_ref?

**Comment by yumingj** [comment]
Yes. In the current implementation, if the reference images are smaller than 512x512, we will do the padding operation to make the reference images have that resolution.
